index.js

```diff
@@ -3,6 +3,8 @@
 module.exports = function implicitFiguresPlugin(md, options) {
   options = options || {};
 
+  var tabIndex = 1;
```

Comment:
This should be inside `function implicitFigures`, to have it reset upon each `md.render()`.